.gitignore

```diff
@@ -1,2 +1,4 @@
-vendor/
-build/
+build
+composer.lock
+phpunit.xml
```

Comment:
The whole point of having a phpunit.xml.dist file, if you can locally create a phpunit.xml locally to override the config for your local dev without having it show up in git.